draft-ietf-quic-recovery.md

```diff
@@ -1030,14 +1030,17 @@ application limited or after a retransmission timeout. In order to limit
 the size of bursts sent into the network, the behavior when restarting from
 idle depends upon whether pacing is used.
 
-If the sender uses pacing, the connection should limit the initial burst of
-packets to no more than the initial congestion window and subsequent packets
-SHOULD be paced. The congestion window does not change while the connection
-is idle.
+If the sender uses pacing, the congestion window MUST not increase and does not
+need to be reduced while the connection is idle.  When restarting from idle, a
+paced connection SHOULD limit the initial burst of packets to no more than the
+initial congestion window and subsequent packets SHOULD be paced.
 
 A sender that does not use pacing SHOULD reset its congestion window to the
-minimum of the current congestion window and the initial congestion window.
-This recommendation is based on Section 4.1 of {{?RFC5681}}.
+smaller of the current congestion window and the initial congestion window, as
+recommended for TCP (see Section 4.1 of {{?RFC5681}}).
+
+A sender MAY implement alternate mechanisms to update its congestion window
+after idle periods, such as those proposed for TCP in {{?RFC7661}}.
```

Comment:
To be clear, we're not really expecting anyone to implement this, right?

Reply:
As formulated this does not permit bursts up to a decayed cwnd because init cwnd is required if not pacing.

Reply:
We do want to limit bursts to init cwnd. Beyond that, instead of describing alt mechanisms here, I think RFC 7661 offers a good and well thought out set of considerations.

Reply:
Right, I suspected as much, just wanted to be sure.